_documentation/user/flags/identify.md

```diff
@@ -5,4 +5,18 @@ type: Good
 summary: Identifies type of nearest flag.
 ---
 
-...
+The Identify (ID) flag allows the player to see into the contents (type) of the nearest flag to the holder.
+
+It manages to identify the nearest flag only if said flag is also in the range specified by the `_identifyRange` server setting, with its argument of distance being expressed in world units.
+
+## Tactics
+
+Identify can be utilized as a means of speeding up the flag choosing process or avoiding of potential bad flags.
+
+Since the tank holding Identify does not have to completely reach the destination flag to check its type, it can shorten the distance the tank has to travel to learn the contents of all the flags, should there be a mass of flags in a single place. In return, doing so can also speed up the process of finding the desired flag.
```

Comment:
```suggestion
Since the tank holding Identify does not have to completely reach the destination flag to check its type, it can shorten the distance the tank has to travel to learn the contents of all the flags, should there be a mass of flags in a single place. Doing so can speed up the process of finding the desired flag.
```

"In return" sounds to me like you're giving up something by using the ID flag.

Reply:
I think the focus of the sentence is wrong. The benefit of ID is less in the distance to be traveled and more in being able to know if you want a flag without having to pick it up. It takes longer to pick up/drop flags (especially bad flags) than to use ID to find your needle in a haystack.

Reply:
@tainn1, I agree with jwmelto on this one. I know it was mentioned in the intro already, but if it could be reiterated, that'd make this ready to merge :+1:

Reply:
@allejo @jwmelto I'm not sure I understand what you mean here. It was mentioned that you can see into the contents of the flag without having to pick it up *and* how it is useful for identifying bad flags. Having to pick it up means getting to it. With ID, you can do more than just avoid picking it up, you can also have a shorter route because you don't have to get next to it.

All in all, I see no point in this argument and on a practical level of usage, even disagree with it. You can push the change should you want two, but I still prefer my way of presentation usability-wise, so that will be on you.

Reply:
@allejo @jwmelto however, what *does* matter is that the flag is, at the end of the day, obtained.

If `_maxFlagGrabs` is set to a low value (especially 1), not having ID results in a gamble of whether the next flag is better than the current from a certain patch of flags. With ID, the best one can be identified without having to risk it teleporting across the map when checking them out.

Reply:
> does not have to completely reach the destination flag to check its type

As discussed in Discord, yea after rereading the latest version it is implied that you don't have to grab the flag to see its type. It is correct, just wasn't obvious to me on my first read-through.

I do like the idea of mentioning that flags can be zapped after one grab, so it's a gamble to grab another flag and hope it's more powerful to you.